e2etests/tests/validate.go

```diff
@@ -126,6 +126,22 @@ func ValidateNodesDoNotHaveRoutes(pods []*v1.Pod, neigh frrcontainer.FRR, prefix
 	})
 }
 
+func ValidateNeighborLocalPrefForPrefix(neigh frrcontainer.FRR, prefix string, expectedLocalPref uint32, ipfam ipfamily.Family) {
+	ginkgo.By(fmt.Sprintf("Checking localPref for prefix %s on neighbor %s", prefix, neigh.Name))
+	Eventually(func() error {
```

Comment:
is the default timeout intentional here? Asking because iirc, in all the other places we force it

Reply:
nope, fixed